hbase_sql/pom.xml

```diff
@@ -1197,6 +1197,7 @@
                                 <argument>-Xdebug</argument>
                                 <!-- <argument>-XX:+UnlockCommercialFeatures</argument>
                                 <argument>-XX:+FlightRecorder</argument> -->
+                                <!--<argument>-Dderby.debug.true=DumpClassFile</argument>-->
```

Comment:
Did you have "DumpClassFile" turned on for debugging?  I think we may want to not have this enabled by default on all systems.

Reply:
This argument has been commented out and I left it here thought this might be something that people want to turn on for debugging.

Reply:
OK, I see.  Thanks.